mypy/test/data.py

```diff
@@ -89,29 +89,34 @@ def parse_test_cases(
 
 
 class DataDrivenTestCase(TestCase):
+    name = None  # type: str
     input = None  # type: List[str]
     output = None  # type: List[str]
 
     file = ''
     line = 0
 
-    perform = None  # type: Callable[['DataDrivenTestCase'], None]
+    perform = None  # type: Callable[..., None]
 
     # (file path, file content) tuples
     files = None  # type: List[Tuple[str, str]]
 
     clean_up = None  # type: List[Tuple[bool, str]]
 
+    is_skip = False
```

Comment:
I'm not quite sure how to read this name. "Is skip" doesn't sound right to me -- I don't generally think of "skip" or "a skip" as a kind of thing that a test or a test case can *be*. Maybe `is_skippable` or `should_skip` or `to_skip`?

Maybe even better would be "marked_skip" -- I see in your `PytestSuite` logic below that there's actually another condition under which you also end up telling pytest to skip the resulting test item. So the tests that "we should skip" or that "are to be skipped" are a little broader than the tests that this attribute is true for.